release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1970,6 +1970,8 @@ In the following tables, features are marked with the following statuses:
 * When you run Cloud-native Network Functions (CNF) latency tests on an {product-title} cluster, the `oslat` test can sometimes return results greater than 20 microseconds. This results in an `oslat` test failure.
 (link:https://issues.redhat.com/browse/RHEL-9279[*RHEL-9279*])
 
+* When installing a cluster on {aws-first} using Local Zones, selecting the `us-east-1-iah-2a` region results in an installation failure. (link:https://issues.redhat.com/browse/OCPBUGS-35538[*OCPBUGS-35538*])
```

Comment:
@bscott-rh 

> installation failure.

install still will be _succeed_, as we have 3 worker nodes in typical AZ.

The issue is _Deploying the edge node into `us-east-1-iah-2a` Local Zone will cause a failure, that the edge node can't be deployed successfully_

Reply:
Thank you for explaining this!